server/lorax_server/adapters/lora.py

```diff
@@ -335,13 +324,13 @@ def load(
                         segment_starts[i] = prefill_head_segment_starts[segment_index]
                         segment_ends[i] = prefill_head_segment_ends[segment_index]
             else:
-                rank_indices = set(indices)
                 batch_indices = [adapter_to_segment[idx] for idx in meta.adapter_indices.tolist()]
-                batch_indices = [idx if idx in rank_indices else -1 for idx in batch_indices]
+                batch_indices = [idx if idx in set(indices) else -1 for idx in batch_indices]
```

Comment:
Oh, yes. I will revert back to the original code. The reason to change it was to not have `rank_indices` variable inside the for loop since the for loop itself loops over another `rank_indices` variable. Maybe, I can rename the `rank_indices` here.